junit-platform-surefire-provider/src/test/java/org/junit/platform/surefire/provider/JUnitPlatformProviderTests.java

```diff
@@ -191,7 +191,7 @@ private void verifyPreconditionViolationException(Map<String, String> properties
 		assertEquals(JUnitPlatformProvider.EXCEPTION_MESSAGE_BOTH_NOT_ALLOWED, throwable.getMessage());
 	}
 
-	private static ProviderParameters providerParametersMock(Class<?>... testClasses) {
+	static ProviderParameters providerParametersMock(Class<?>... testClasses) {
```

Comment:
Considering that this method is now used in both `JupiterPlatformProviderTests` and `LegacyReportingTests`, I suggest either creating a superclass containing this method which both classes will extend from, or to create a tiny utilities class which both classes will import.